.github/workflows/deploy.yml

```
@@ -20,6 +20,8 @@ jobs:
         run: |
           python -m pip install cibuildwheel==2.2.2
       - name: Build wheels
+        env:
+          AER_CMAKE_OPENMP_BUILD: 1
```

Do you think it would be better to set this via `CIBW_ENVIRONMENT`: https://cibuildwheel.readthedocs.io/en/stable/options/#environment so it gets set in the build environment directly? I think this is working because macOS builds don't use any virtualization and run natively on the host but I feel like leveraging the CIBW env variable to set this is probably more future proof.

So probably something like:

```suggestion
        env:
         CIBW_ENVIRONMENT: AER_CMAKE_OPENMP_BUILD=1
```